package-list.json

```diff
@@ -7,10 +7,10 @@
       {
         "version": "current",
         "desc": "Continuous Integration Build (latest in version control)",
-        "path": "http://hl7.org/fhir/uv/fhircast",
+        "path": "http://hl7.org/fhir/uv/fhircast/build",
         "status": "ci-build",
         "sequence": "STU3",
-        "fhirversion": "4.0.1",
+        "fhirversion": "4.3.0",
```

Comment:
Would this now be "5.0.0"?

Reply:
Has anyone ever tested R4B resources with FHIRcast? Why shouldn't we keep this at 4? (Like until folks begin supporting R6?)

Reply:
4.3.0 is required for SubscriptionTopic.